enterprise/libs.fabric8/nbproject/project.properties

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+is.autoload=true
+javac.compilerargs=-Xlint -Xlint:-serial
+javac.release=11
+sigtest.gen.fail.on.error=false
+release.external/kubernetes-client-6.7.0.jar=modules/ext/kubernetes-client-6.7.0.jar
+release.external/kubernetes-client-api-6.7.0.jar=modules/ext/kubernetes-client-api-6.7.0.jar
+release.external/kubernetes-model-autoscaling-6.7.0.jar=modules/ext/kubernetes-model-autoscaling-6.7.0.jar
+release.external/kubernetes-model-events-6.7.0.jar=modules/ext/kubernetes-model-events-6.7.0.jar
+release.external/kubernetes-model-rbac-6.7.0.jar=modules/ext/kubernetes-model-rbac-6.7.0.jar
+release.external/kubernetes-model-batch-6.7.0.jar=modules/ext/kubernetes-model-batch-6.7.0.jar
+release.external/kubernetes-model-extensions-6.7.0.jar=modules/ext/kubernetes-model-extensions-6.7.0.jar
+release.external/kubernetes-model-resource-6.7.0.jar=modules/ext/kubernetes-model-resource-6.7.0.jar
+release.external/kubernetes-client-6.7.0.jar=modules/ext/kubernetes-client-6.7.0.jar
+release.external/kubernetes-model-certificates-6.7.0.jar=modules/ext/kubernetes-model-certificates-6.7.0.jar
+release.external/kubernetes-model-flowcontrol-6.7.0.jar=modules/ext/kubernetes-model-flowcontrol-6.7.0.jar
+release.external/kubernetes-model-scheduling-6.7.0.jar=modules/ext/kubernetes-model-scheduling-6.7.0.jar
+release.external/kubernetes-client-api-6.7.0.jar=modules/ext/kubernetes-client-api-6.7.0.jar
+release.external/kubernetes-model-common-6.7.0.jar=modules/ext/kubernetes-model-common-6.7.0.jar
+release.external/kubernetes-model-metrics-6.7.0.jar=modules/ext/kubernetes-model-metrics-6.7.0.jar
+release.external/kubernetes-model-storageclass-6.7.0.jar=modules/ext/kubernetes-model-storageclass-6.7.0.jar
+release.external/kubernetes-model-admissionregistration-6.7.0.jar=modules/ext/kubernetes-model-admissionregistration-6.7.0.jar
+release.external/kubernetes-model-coordination-6.7.0.jar=modules/ext/kubernetes-model-coordination-6.7.0.jar
+release.external/kubernetes-model-networking-6.7.0.jar=modules/ext/kubernetes-model-networking-6.7.0.jar
+release.external/snakeyaml-2.0.jar=modules/ext/snakeyaml-2.0.jar
```

Comment:
`project.xml` says this `libs.fabric8` depends on `snakeyaml_engine` -- is this local jar required (e.g. an older version, does not work with snakeyaml_engine impl) ?